libs/checkout/src/payment/containers/payment.component.spec.ts

```diff
@@ -10,18 +11,16 @@ import { PaymentInfo } from '../../models/payment/payment-info';
 describe('PaymentContainer', () => {
   let component: PaymentContainer;
   let fixture: ComponentFixture<PaymentContainer>;
-  let store;
+  let store: MockStore<any>;
   let initialPaymentInfo: PaymentInfo;
   const paymentFactory: DaffPaymentFactory = new DaffPaymentFactory();
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [
-        StoreModule.forRoot({
-          payments: combineReducers(fromPayment.reducers),
-        })
-      ],
-      declarations: [ PaymentContainer ]
+      declarations: [ PaymentContainer ],
```

Comment:
`PaymentContainer` on new line.

Reply:
Since this doesn't currently fail linting, I'm going to ignore it. If we want to enforce these rules, open a new issue and we can address at a later time.